app/graphql/types/rock_and_pebble_type.rb

```diff
@@ -1,20 +1,20 @@
 module Types
-    class RockAndPebbleType < Types::BaseObject
-        field :id, ID, null: false
-        field :rock, Types::UserType, null: false
-        field :pebble, Types::UserType, null: false
-        field :active, Boolean, null: false
-        field :my_rocks, [Types::UserType], null: true
-        field :my_pebbles, [Types::UserType], null: true
+  class RockAndPebbleType < Types::BaseObject
+      field :id, ID, null: false
+      field :rock, Types::UserType, null: true
+      field :pebble, Types::UserType, null: true
+      field :active, Boolean, null: false
+      field :my_rocks, [Types::UserType], null: true
+      field :my_pebbles, [Types::UserType], null: true
 
-        def my_rocks
-          user = User.find(object.pebble_id)
-          user.rocks
-        end
+  def my_rocks
+    user = User.find(object.pebble_id)
+    user.rocks
+  end
 
-        def my_pebbles
-          user = User.find(object.rock_id)
-          user.pebbles
-        end
-    end
+  def my_pebbles
+    user = User.find(object.rock_id)
+    user.pebbles
+  end
+  end
```

Comment:
I think this is git formatting. First end is for the def my_pebbles, end after is for Class RockAndPebble... and third and last end is for the module Types. Otherwise, it looks correct indentation wise in my text reader.

Reply:
Ahh got it. I switched over to viewing the file and I can see it now. Thanks for clarifying!